dj/sql/functions.py

```diff
@@ -1277,6 +1277,25 @@ def infer_type(  # noqa: F811
     return ct.ListType(element_type=element_type)
 
 
+class Array_Agg(Function):  # pylint: disable=abstract-method,disable=invalid-name
```

Comment:
We can probably just use `ArrayAgg` for consistency (or switch everything else to be snake rather than camel case).